packages/WebVideo/WebVideo.jsx

```diff
@@ -46,11 +46,13 @@ const WebVideo = ({
   beginMuted,
   videoLength,
   copy,
+  onStart,
   ...rest
 }) => {
   const [started, setStarted] = useState(false)
 
   const initializeYoutubePlayer = event => {
+    onStart()
```

Comment:
If `onStart` is `undefined`, I'm not sure if this will throw an error.

Reply:
It doesn't, but I changed the default to `()=>{}` anyway since it's more proper.